ajax/libs/mapbox-gl/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "mapbox-gl",
+  "description": "A WebGL interactive maps library",
+  "version": "0.32.1",
+  "main": "src/index.js",
+  "filename": "src/index.min.js",
```

Comment:
`filename` should point to the minified main file. In this case, it should point to `mapbox-gl.js`.